samples/tutorials/mnist/00_Kubeflow_Cluster_Setup.ipynb

```diff
@@ -0,0 +1,392 @@
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#      http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License.\n",
+    "# =============================================================================="
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Enviroment Setup\n",
+    "This notebook helps you deploy kubeflow cluster, and necessary setup for running tutorial from different environment\n",
+    "- Notebook server outside kubeflow cluster\n",
+    "- Notebook server on AI platform\n",
+    "- Notebook server within kubeflow cluster"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "### Pre-condition\n",
+    "- You have a [GCP project setup](https://www.kubeflow.org/docs/gke/deploy/project-setup/) for your Kubeflow Deployment with you having the [owner role](https://cloud.google.com/iam/docs/understanding-roles#primitive_role_definitions) for the project and with the following APIs enabled:\n",
+    "    - [Compute Engine API](https://pantheon.corp.google.com/apis/library/compute.googleapis.com)\n",
+    "    - [Kubernetes Engine API](https://pantheon.corp.google.com/apis/library/container.googleapis.com)\n",
+    "    - [Identity and Access Management(IAM) API](https://pantheon.corp.google.com/apis/library/iam.googleapis.com)\n",
+    "    - [Deployment Manager API](https://pantheon.corp.google.com/apis/library/deploymentmanager.googleapis.com)\n",
+    "    - [Cloud Resource Manager API](https://pantheon.corp.google.com/apis/library/cloudresourcemanager.googleapis.com)\n",
+    "    - [Cloud Filestore API](https://pantheon.corp.google.com/apis/library/file.googleapis.com)\n",
+    "    - [AI Platform Training & Prediction API](https://pantheon.corp.google.com/apis/library/ml.googleapis.com)\n",
+    "- You have set up [OAuth for Cloud IAP](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/)\n",
+    "- You have installed and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)\n",
+    "- You have installed [gcloud-sdk](https://cloud.google.com/sdk/)\n",
+    "\n",
+    "### Setup steps\n",
+    "* Deploy a kubeflow cluster through [CLI](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/)\n",
+    "    - `NOTE` : The [UI](https://www.kubeflow.org/docs/gke/deploy/deploy-ui/) method of deploying a kubeflow cluster does not support Kubeflow v0.7 yet\n",
```

Comment:
I checked that the one-click supports 0.7.0, could you verify?

Reply:
Yes indeed. For the time of creating this tutorial, 0.7.0 is not there yet. Modified accordingly.